migrate_pim/index.rst

```diff
@@ -163,10 +177,10 @@ Then, follow the migration guides located in your archive to upgrade your projec
 * From EE v1.0 to EE v1.3: UPGRADE-1.3.md
 
 
-Major Version
--------------
+How to upgrade to a major version?
+----------------------------------
 
-We release major versions every year, these versions bring new features and larger changes to answer to growing needs.
+We release a major version each year, these new major version brings new features and larger changes to answer clients growing needs.
```

Comment:
**this** new major version

Reply:
or these new major versions bring

Reply:
went for "these new major versions bring" thanks!